src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -52,12 +72,14 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
       pathPrefix("workspaces") {
         pathEnd {
           requireUserInfo() { _ =>
-            passthrough(rawlsWorkspacesRoot, HttpMethods.GET)
+            mapHttpResponse(transformListWorkspaceRequest) {
```

Comment:
This (and the singular GET) are not pure passthrough methods. They interpret the realm from Rawls into the isProtected property.